test/contracts/consumer/fabric8auth/api_token.go

```diff
@@ -0,0 +1,55 @@
+package fabric8auth
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"testing"
+
+	"github.com/fabric8-services/fabric8-auth/test/contracts/model"
+	"github.com/pact-foundation/pact-go/dsl"
+)
+
+// AuthAPITokenKeys defines contract of /api/status endpoint
+func AuthAPITokenKeys(t *testing.T, pact *dsl.Pact) {
+
+	log.Printf("Invoking AuthAPITokenKeys now\n")
+
+	// Pass in test case
+	var test = func() error {
```

Comment:
Why not outsource this method? You've defined it twice up to this point and peeking down the code, it looks as if you're going to redefine the same method at least once more with only changing the URL.

Reply:
Thanks for the tip - fixed.